drivers/python/rethinkdb/_export.py

```diff
@@ -298,14 +306,15 @@ def csv_writer(filename, fields, delimiter, task_queue, error_queue):
             pass
 
 def launch_writer(format, directory, db, table, fields, delimiter, task_queue, error_queue):
-    if format == "json":
+    if format == "json" or format == "nsj":
         filename = directory + "/%s/%s.json" % (db, table)
```

Comment:
I think the file extension should reflect the data type - since the file can't be parsed as valid JSON, it shouldn't be named with a `.json`.